app.js

```diff
@@ -110,6 +110,7 @@ function renderCurrentGeneration() {
   newGameGrid.id = 'game-grid';
   newGameGrid.addEventListener('mousedown', handleCellClick);
 
+  //possibly a good place for more funciton-interior comments
```

Comment:
One thing I think would improve comprehension to the uninitiated would be more comments inside of functions detailing what's going on. 

It's not as pretty, but there were def funcs with long blocks of code that could use it (although in some of your functions the JSDoc comments were thorough).